### DISCUSSION TITLE
Bilibili. tv

### DISCUSSION THREAD

**Discussion by Budhi108** [aclap-dev/video-downloadhelper#703] [Help and Q&A]
### Your question

Hi there,

Bilibili is on the list of sites that support downloads, but VDH doesn't show anything when the video starts.

here is the link of one episode:

https://www.bilibili.tv/en/video/4791408235642880

### Addon & CoApp version and details
Video DownloadHelper
Version 9.0.1.22 (stable)
Browser locale: hr
Production build
Built on 2024-05-24
Build options: browser=chrome, linuxlic=false, noyt=true, target=google

Found companion app: VdhCoApp 2.0.19
Companion app binary: C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by Mattmjs** [comment]
I looked at this and it seems that the stream information is in a .bin file. What is a .bin ? I don't know.
This may be why it doesn't detect the video. I decided to see if I could open the m4s links and one opened the video without audio and another opened the audio. You could get the media like that and use a software to merge the 2 together.

![Screenshot 2024-05-27 181935](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/7c402029-ebf8-4ebb-a2f2-9688705ea1aa)

**Comment by Budhi108** [comment]
thank you for your effort. I found the same video on another site that works with VDH. but maybe staff should remove bilibili.tv from supported sites if VDH is not able to download video from there

**Reply by Mattmjs** [reply]
Well I'm a user like you, but I wonder what the dev thinks of this site whether it could work or not.
